_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,133 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Notifier can send a variety of notifications to alert you
+of Forseti events. You can configure notifications to send to different
```

Comment:
Notifier can dispatch a variety of messages through various channels and varying formats alerting you to events in your environment.

The second sentence can be dropped.

Reply:
Done